tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -443,7 +443,7 @@ public Response getCompensatedCount() {
     @GET
     @Path("/cancelOn")
     @Produces(MediaType.APPLICATION_JSON)
-    @LRA(value = LRA.Type.REQUIRED, cancelOn = {Response.Status.NOT_FOUND, Response.Status.BAD_REQUEST})
+    @LRA(value = LRA.Type.REQUIRED, cancelOn = {Response.Status.NOT_FOUND, Response.Status.BAD_REQUEST}, terminal = false)
```

Comment:
`terminal` doesn't matter here I think since we are always returning a Response which leads to Cancel

Reply:
+1